.idea/ant.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project version="5">
+<project version="4">
```

Comment:
Are these changes to the IntelliJ IDEA ant build configuration intended?
It seems odd to be dropping down a version number !
Ta, Steve.